apps/openmw/mwmechanics/mechanicsmanagerimp.cpp

```diff
@@ -1366,11 +1396,12 @@ namespace MWMechanics
 
         const MWMechanics::NpcStats& victimStats = victim.getClass().getNpcStats(victim);
 
-        // Simple check for who attacked first: if the player attacked first, a crimeId should be set
+        // Simple check for who attacked first:
+        // if the player or his follower/summon attacked first, a crimeId should be set.
         // Doesn't handle possible edge case where no one reported the assault, but in such a case,
         // for bystanders it is not possible to tell who attacked first, anyway.
         if (victimStats.getCrimeId() != -1)
-            commitCrime(attacker, victim, MWBase::MechanicsManager::OT_Murder);
+            commitCrime(attackerIsPlayerFollowerOrSummon ? player : attacker, victim, MWBase::MechanicsManager::OT_Murder);
```

Comment:
This doesn't seem to be in line with the original game. Try this:
<pre>
npc1->aifollow, player, 0,0,0,0
npc1->startcombat, npc2
</pre>
No bounty is added.